### DISCUSSION TITLE
PHP IntelliSense

### DISCUSSION THREAD

**Discussion by sgt-spike** [coder/code-server#5034] [Q&A]
Hello,
Nice server.  My organization will not allow us to use VScode so I set this up on my home server to access it across the web. 

I have installed the extension PHP IntelliSense by felixbecker.  The ext recommands installing php for the ext to work.  Where would be the best location for the php executables?

**Comment by jsjoeio** [comment]
Great question! How are you running code-server? (What OS, setup, etc.)

**Reply by sgt-spike** [reply]
> Great question! How are you running code-server? (What OS, setup, etc.)

I have it running in docker on an UnRaid server.  I have it served through a SWAG proxy server

**Reply by jsjoeio** [reply]
Hmm...I am not familiar with UnRaid servers or SWAG proxy servers. 

Could you install PHP as part of your Docker image? [Here's an example](https://github.com/jsjoeio/joe-image/blob/main/Dockerfile#L81-L84) of how I install Deno for instance in a Docker-based setup I use in case you use it as inspiration.
